src/middleware.coffee

```diff
@@ -21,6 +21,7 @@ class Middleware
   # Returns nothing
   # Returns before executing any middleware
   execute: (context, next, done) ->
+    done ||= ->
```

Comment:
I thought it was a rubyism for some reason.

From http://coffeescript.org/#operators:

> It's a little difficult to check for the existence of a variable in JavaScript. if (variable) ... comes close, but fails for zero, the empty string, and false. CoffeeScript's existential operator ? returns true unless a variable is null or undefined, which makes it analogous to Ruby's nil?
>
>It can also be used for safer conditional assignment than ||= provides, for cases where you may be handling numbers or strings.